src/cli.ts

```diff
@@ -155,24 +155,6 @@ export async function cli(program: Command) {
         await withdraw_getHash(ctx, options.to, options.amount, options.transactionId, options.nonce)
       }
     })
-  // ledger two-step manual signing

```

Comment:
Please add this block back, we are using these 2 options for ledger signing of withdrawal transactions

Reply:
@vickypanwar16